concordium-consensus/src/Concordium/Skov/Update.hs

```diff
@@ -481,8 +481,10 @@ doStoreBlock pb@GB.PendingBlock{..} = unlessShutDown $ do
 doReceiveTransaction :: (TreeStateMonad pv m, TimeMonad m, SkovQueryMonad pv m) => BlockItem -> Slot -> m UpdateResult
 doReceiveTransaction tr slot = unlessShutDown $ do
   -- Don't accept the transaction if its expiry time is too far in the future
-  expiryTooLate <- isExpiryTooLate
+  now <- currentTime
+  expiryTooLate <- isExpiryTooLate now
   if expiryTooLate then return ResultExpiryTooLate
```

Comment:
```suggestion
  expiryTooLate <- isExpiryTooLate now
  maxTimeToExpiry <- rpMaxTimeToExpiry <$> getRuntimeParameters
  if msgExpiry tr > maxTimeToExpiry + utcTimeToTransactionTime now then return ResultExpiryTooLate
```
Removing the `where` clause.

Reply:
I like that the function is named so I do not agree it should be inlined.